WinUIGallery/Samples/ControlPages/Design/IconographyPage.xaml

```diff
@@ -246,15 +251,15 @@
                                         AutomationProperties.Name="{x:Bind}"
                                         CornerRadius="12">
                                         <Grid
-                                            Padding="8,3,8,4"
+                                            Padding="8,0"
+                                            Height="24"
```

Comment:
We shouldn't hardcode the height here, because I'm not sure what's going to happen when text scaling is > 100%. Please revert

Reply:
> We shouldn't hardcode the height here

I tweaked things because the corner radius is hardcoded at 12. So I added height=24 to make sure it always matches the corner. The padding at the top/bottom can only cut into the text.

Reply:
I don't think WinUI have a hardcoded height for the reason that things need to scale (e.g. see button padding). So let's stick to that.

Reply:
If both height _and_ vertical padding are removed, the corners get messed up. What I think is that the **padding** was used to fix this. I think the best compromise is no vertical padding and `MinHeight="24"`.

Reply:
Off-topic: try setting text size to something extremely large, like 175% or 200%. Note that the content of the SidePanel can get clipped at the bottom. This is because the **SampleThemeListener** has `MaxHeight="600"`.

Reply:
Alright, let's stick to with the MinHeight 👍.

```<ScrollViewer Padding="16,16,8,16">```

We should make this scrollable - we can wrap the with a `ScrollViewer` and that should fix it :)

Reply:
> We should make this scrollable

Let's create a new issue or PR for this, because we already have a scroller on the page _and_ a scroller in the **IconsItemsView** and this is already a bit "finniky".